src/components/wallet-card.tsx

```diff
@@ -73,6 +73,20 @@ export const WalletCard = memo(
     const screenWidth = useWindowDimensions().width;
 
     const protectionStatus = useMemo(() => {
+      // Ledger and Hot always has Full Protection
+      if ([WalletType.ledgerBt, WalletType.hot].includes(wallet.type)) {
+        return ProtectionStatus.full;
+      }
+
+      // Hot only need socialLinkEnabled
+      if (wallet.type === WalletType.hot) {
+        if (!wallet.socialLinkEnabled) {
```

Comment:
wallet.socialLinkEnabled always false for WalletType.hot

Reply:
Fixed